tspng/extraction.py

```diff
@@ -5,7 +5,7 @@
 import json
 import os
 
-def extract(path,mime_type=MIME_TYPE):
+def extract_from_file(path,mime_type=MIME_TYPE):
```

Comment:
I would add a space after the comma between the two arguments, `path,mime_type`.

Reply:
Made the change as requested